bpf/lib/nat.h

```diff
@@ -782,9 +782,8 @@ snat_v4_nat_handle_icmp_error(struct __ctx_buff *ctx, __u64 off,
 	__u32 icmpoff;
```

Comment:
*(On the commit description.)*

I'd consider that a fix on the original implementation, so could you add a `Fixes:` tag referencing the commit you fixed here. I think you can also drop the reference to the original pull request from your description. It's best to just refer to the commits.

The description is a bit hard to follow, I think because of a mix of past and present tenses. I'd suggest to refer to the previous commit in the past tense and to your change in the present tense. So for example, the first sentence would become:
```
Commit xxxx ("...") added handling procedure...
```

Finally, could you also wrap lines at 75 characters in the commit description?